packages/components/socialFeed/SocialActions/DislikeButton.tsx

```diff
@@ -17,16 +17,16 @@ export const DislikeButton: FC<{
   post: Post;
   setPost: Dispatch<SetStateAction<Post>>;
 }> = ({ post, setPost }) => {
-  const { handleReaction, isPostMutationLoading } = useSocialReactions({
+  const { handleReact, isReactLoading } = useSocialReactions({
     post,
     setPost,
   });
 
-  if (isPostMutationLoading)
+  if (isReactLoading)
     return <ActivityIndicator animating color={secondaryColor} size={32} />;
   return (
     <TouchableOpacity
-      onPress={() => handleReaction(DISLIKE_EMOJI)}
+      onPress={() => handleReact(DISLIKE_EMOJI)}
```

Comment:
why use `react` insteand of `reaction`.
maybe for new user, at first look it could be confusing with React the library 
Reaction feel more explicit isn't

Reply:
It was confusing, I felt like "handleReaction" meant "It fetches reaction". Because the data is named "reactions", so I prefered to use the verb "react", because this hook adds a reaction (the user reacts to a post), and it doesn't fetch data.
But it's not the scope, so I revert this change !
Maybe I'll open a new PR to rename the hook and the function to "useReactToPost" and "reactToPost".

So => https://github.com/TERITORI/teritori-dapp/pull/1393/commits/d9b30c8e9ab5f051c20dc27c71bef2be4ec4cc7f